syntax tree implementation

millenniumbismay opened the pull request.

millenniumbismay committed:
Create Running Github from terminal

millenniumbismay committed:
Add files via upload

millenniumbismay committed:
Merge pull request #1 from itsmb16/stemmed-words-&-syntax-tree-implementation

Add files via upload

ujjaldas1997 commented on data/Running Github from terminal:
```diff
@@ -0,0 +1,24 @@
+      #This will Merge the changes from upstream/master into your local master branch. 
+      This brings your fork's master branch into sync with the upstream repository, without losing your local changes.
+$ git remote -v #This will show all the information about your current repositories as well as the master repository in the local system
+$ git push origin master #This will update your own repository with any changes made on the local directrory.
```
not needed actually.

ujjaldas1997 replied:
Delete data/Running Github from terminal

ujjaldas1997 commented:
Clean all the output files.

millenniumbismay committed:
Delete stemming.ipynb

millenniumbismay committed:
Delete stemmedwords.txt

millenniumbismay committed:
Delete englishwords.txt

millenniumbismay committed:
Delete eng_odia_dict.txt

millenniumbismay committed:
Delete eng_odia.txt

millenniumbismay committed:
Update syntax_tree.py

It dumps the english words to odia meanings(separated in case a word has more than a odia meaning) and its POS tag
It has been implemented using the syntax tree data structure for python
So the output is not in human readable format
There is a file  eng_odia.txt which has a human readable format of this output.
But this kind of dumping helps in very fast retrieval and thus accessing time 
is very less making it viable for machine translation.

millenniumbismay committed:
Add files via upload

This file has the human readable format for english to odiya dictionary. It is not dumped but is just written into the file.

millenniumbismay committed:
Delete stemming.py

ujjaldas1997 committed:
Delete Running Github from terminal
